.github/workflows/pull.yml

```diff
@@ -406,7 +406,7 @@ jobs:
         output=$(ls -la cmake-out/test/size_test)
         arr=($output)
         size=${arr[4]}
-        threshold="47560"
+        threshold="51720"
```

Comment:
Similar to above, note that the "true" size increase is around ~64 bytes. See https://github.com/pytorch/executorch/pull/11217/files.